### PR TITLE
Linting fixes

### PR DESCRIPTION
**Pull request by danlugo92**

### CODE REVIEW

**Comment by danlugo92** [comment]
Tested ✅

File: src/pages/QRScanner/index.tsx [hunk 1]
```diff
@@ -33,8 +31,8 @@ const QRScanner = () => {
   const [memo, setMemo] = useState("");
   const [LNURLdata, setLNURLdata] = useState(null);
   const [scanQR, setScanQR] = useState(true);
-  const [done, setDone] = useState(null);
-  const [error, setError] = useState(null);
+  const [, setDone] = useState(null);
```
Comment (Emad-salah) [inline src/pages/QRScanner/index.tsx:34]: if these states are unused, can you remove them entirely just to get rid of any additional unnecessary re-renders?

File: src/pages/Send/index.tsx [hunk 2]
```diff
@@ -34,7 +33,7 @@ const SendPage = () => {
   const [unit, setUnit] = useState("sats");
   const [contact, setContact] = useState<properties>({});
   const [error, setError] = useState("");
-  const [loading, setLoading] = useState(false);
+  const [, setLoading] = useState(false);
```
Comment (Emad-salah) [inline src/pages/Send/index.tsx:36]: Same as above

File: src/common/Post/components/Stream/index.tsx [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
-  postId,
   tipValue,
   tipCounter,
   hideRibbon,
   width,
-  timeout = 10000
 }) => {
   const playerDOM = useRef(null);
-  const [isLive, setIsLive] = useState(false);
+  const [isLive] = useState(false);
```
Comment (Emad-salah) [inline src/common/Post/components/Stream/index.tsx:14]: if setIsLive is unused then `isLive` is probably _always_ false, can you get rid of that too?

**Review by Emad-salah — CHANGES_REQUESTED** [review]
I've left some comments regarding some minor changes, otherwise, everything looks good 🙂

File: src/common/Post/components/Stream/index.tsx (hunk 3 above)
Comment (danlugo92) [reply src/common/Post/components/Stream/index.tsx:15]: done

File: src/pages/Send/index.tsx (hunk 2 above)
Comment (danlugo92) [reply src/pages/Send/index.tsx:36]: done

**Pull request merged**